keyboards/kikoslab/kl90/keymaps/via/keymap.c

```diff
@@ -14,28 +14,35 @@ GNU General Public License for more details.
 You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
+#include "config.h"
```

Comment:
```suggestion
```

Reply:
Hey just for clarity, how would you prefer I go about updating the encoder map key delay? The QMK docs mention using a config.h to define the ENCODER_MAP_KEY_DELAY value which was what I was intending to do here.

I'm happy to go a different route, just not sure which.

Reply:
`config.h` file is automatically included internally.

Reply:
Ah gotcha, thank you for the clarification. Will remove.